rest-api/data/awardees.csv

```diff
@@ -0,0 +1,26 @@
+Awardee ID,Name,Type

```

Comment:
Double-checking - the contents of all of these lists are public information, right?

Reply:
Correct.